### ISSUE TITLE
A11y_SQLAzureDataStudio_ExportBacpac_ExportBacpacSettings_ScreenReader: The required combo-box is not announced as required

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#9261]
[“Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

#A11y_SQLAzureDataStudio; #A11yMAS; #A11ySev2; #A11yTCS; #SQL Azure Data Studio; #Benchmark; #DesktopApp; #MacOs; #MAS1.3.1; #VoiceOver; #MAS3.3.2;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.15.0-insider
Commit: 3a8b74a
Date: 2020-02-11T05:49:41.181Z
VS Code: 1.42.0
Electron: 7.1.11
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Darwin x64 19.3.0
Mac OS:10.15.3

**Additional Details:**
Screen reader: Voiceover
Mac OS:10.15.3
MAS References:MAS1.3.1, MAS3.3.2

**Repro Steps:**
1. Launch Azure Data Studio application.
2. Connect to server with windows credentials.
3. Navigate to Extensions from left menu.
4. Search & install Dacpac extension.
5. Navigate to Connections from left menu.
6. Expand the server connected in step 2.
7. Right click an on prem database and select Data-tier application wizard from Context menu.
8. Select forth radio button.
9. Navigate till next button & hit enter.
10. Launch Voiceover and tab to the first combo-box.

**Actual:**
On navigating to the first combo-box "source server" screen reader does not annouce it as required which is visually seen on screen. 
Simmilarly for "source database"

**Similar isue is the following page:**
1.Import Bacpac Settings

**Expected:**
When navigated to the combo-box with tab key the name of the combo-box the item name as well as the instrcution as required must be announced.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets: https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
Screen readers users will not know that field is required.

**MAS Reference:**
MAS 1.3.1:https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}
MAS3.3.2:https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={4b95a70d-258c-474a-820e-5c8b79d5c0bf}

**Attachment link for Reference :**
https://microsoft-my.sharepoint.com/:p:/g/personal/v-stfe_microsoft_com1/ETGCKDliVNtPgqczVRnjQycBE2078o5vk_OKm5xfBY9DRw?e=bRjAIh

**Comment by kisantia** [comment]
Duplicate of #9183. Fixed by #9200

**Comment by BhanuPrakashGit** [comment]
Verified the bug and does not repro.
GithubTags:#Closed;
PFA for reference
[9261_Closed.zip](https://github.com/microsoft/azuredatastudio/files/10732322/9261_Closed.zip)

**Comment by mstechie** [comment]
GitHubTags:#SQLADS-MAC-Feb2020;
